src/GitHub.App/Factories/HostCacheFactory.cs

```diff
@@ -39,5 +39,23 @@ public IBlobCache Create(HostAddress hostAddress)
 
         IOperatingSystem OperatingSystem { get { return operatingSystem.Value; } }
         IBlobCacheFactory BlobCacheFactory { get { return blobCacheFactory.Value; } }
+
+        bool disposed;
+        protected virtual void Dispose(bool disposing)
```

Comment:
Given that this class doesn't have a finalizer, is it even necessary to have a `Dispose` that takes in `disposing` since it's always true? I guess a derived class could add a finalizer, but we could solve that by making this sealed. I don't feel too strongly about it though.